numba/cuda/tests/cudadrv/test_mvc.py

```diff
@@ -0,0 +1,52 @@
+import multiprocessing as mp
+import traceback
+from numba.cuda.testing import unittest, CUDATestCase
+from numba.cuda.testing import skip_on_cudasim, skip_under_cuda_memcheck
+from numba.tests.support import linux_only
+
+
+def child_test():
+    from numba import config, cuda
+
+    # Change the MVC config after importing numba.cuda
+    config.CUDA_ENABLE_MINOR_VERSION_COMPATIBILITY = 1
+
+    @cuda.jit
+    def f():
+        pass
+
+    f[1, 1]()
+
+
+def child_test_wrapper(result_queue):
+    try:
+        output = child_test()
+        success = True
+    # Catch anything raised so it can be propagated
+    except: # noqa: E722
+        output = traceback.format_exc()
+        success = False
+
+    result_queue.put((success, output))
+
+
+@linux_only
+@skip_under_cuda_memcheck('May hang CUDA memcheck')
+@skip_on_cudasim('Simulator does not require or implement MVC')
+class TestMinorVersionCompatibility(CUDATestCase):
+    def test_mvc(self):
```

Comment:
Confirming this fails on `main`:

```
======================================================================
FAIL: test_mvc (numba.cuda.tests.cudadrv.test_mvc.TestMinorVersionCompatibility)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/gmarkall/numbadev/numba/numba/cuda/tests/cudadrv/test_mvc.py", line 48, in test_mvc
    self.fail(output)
AssertionError: Traceback (most recent call last):
  File "/home/gmarkall/numbadev/numba/numba/cuda/tests/cudadrv/test_mvc.py", line 23, in child_test_wrapper
    output = child_test()
  File "/home/gmarkall/numbadev/numba/numba/cuda/tests/cudadrv/test_mvc.py", line 18, in child_test
    f[1, 1]()
  File "/home/gmarkall/numbadev/numba/numba/cuda/dispatcher.py", line 541, in __call__
    return self.dispatcher.call(args, self.griddim, self.blockdim,
  File "/home/gmarkall/numbadev/numba/numba/cuda/dispatcher.py", line 675, in call
    kernel = _dispatcher.Dispatcher._cuda_call(self, *args)
  File "/home/gmarkall/numbadev/numba/numba/cuda/dispatcher.py", line 683, in _compile_for_args
    return self.compile(tuple(argtypes))
  File "/home/gmarkall/numbadev/numba/numba/cuda/dispatcher.py", line 928, in compile
    kernel.bind()
  File "/home/gmarkall/numbadev/numba/numba/cuda/dispatcher.py", line 207, in bind
    self._codelibrary.get_cufunc()
  File "/home/gmarkall/numbadev/numba/numba/cuda/codegen.py", line 184, in get_cufunc
    cubin = self.get_cubin(cc=device.compute_capability)
  File "/home/gmarkall/numbadev/numba/numba/cuda/codegen.py", line 155, in get_cubin
    linker = driver.Linker.new(max_registers=self._max_registers, cc=cc)
  File "/home/gmarkall/numbadev/numba/numba/cuda/cudadrv/driver.py", line 2612, in new
    return MVCLinker(max_registers, lineinfo, cc)
  File "/home/gmarkall/numbadev/numba/numba/cuda/cudadrv/driver.py", line 2692, in __init__
    self._linker = CubinLinker(f"--arch={arch}")
NameError: name 'CubinLinker' is not defined
```

and passes locally on this branch for me, so the test appears to be sound.